app/jobs/include_publisher_in_payout_report_job.rb

```diff
@@ -35,15 +39,11 @@ def perform(arguments = {})
     when UPHOLD
       potential_payment_job = Payout::UpholdService
     when BITFLYER
-      potential_payment_job = Payout::BitflyerService
+      return Payout::BitflyerService.build.perform(attrs_payment_job)
     when MANUAL
       potential_payment_job = Payout::ManualPayoutReportPublisherIncluder
     end
 
-    potential_payment_job.new(
-      publisher: publisher,
-      payout_report: payout_report,
-      should_send_notifications: should_send_notifications,
-    ).perform
+    potential_payment_job.new(attrs_payment_job).perform
```

Comment:
Would you restore using keyword arguments? That way the caller knows what's going on

Reply:
Sure, I'll put it back. I used a var since I need to use it twice, but if you prefer the readability of the other, I'm not married to it.